### ISSUE TITLE
Kick off

### ISSUE THREAD

**Issue by randomC0der** [PhoenixKarton/Meme-Bot#4]

**Comment by PhoenixKarton** [comment]
Hast du gerade nichts zu tun?

**Comment by randomC0der** [comment]
Doch, aber ich bin hier noch eingeloggt gewesen wegen DS und da habe ich die unread notifications gesehen. Ich habe das Ticket als Erinnerung angelegt :) Ich bin auf github, weil ich https://github.com/Http-Multipart-Data-Parser/Http-Multipart-Data-Parser gefunden habe, was ich brauche, um meine SOAP Responses richtig zu lesen
